const { catchAsyncError } = require("./catchAsyncError");
const ResError = require('./ResError');

module.exports = {
    catchAsyncError,
    ResError,
};